lab2.js

```diff
@@ -71,48 +123,82 @@ assert(blob.hoursToOoze(1000, 1) === hoursSpentInDowington,
 // TODO: write three more assertions like the two above, testing out
 // the hoursToOoze method.
 
+assert(blob.hoursToOoze(10, 1) === 3, '10 people takes the blob 3 hours to consume');
+assert(blob.hoursToOoze(100, 1) === 8, '100 people takes the blob 8 hours to consume');
+assert(blob.hoursToOoze(500, 1) === 14, '500 people takes the blob 14 hours to consume.');
+assert(blob.hoursToOoze(500, 0) === 0, '0 people consumed per hour means no time needed.');
+
+// END SECOND PART OF PROBLEM 1
+
 //*********************************************************
 // PROBLEM 2: Universal Translator. 20 points
 //*********************************************************
-
 var hello = {
   klingon: 'nuqneH',  // home planet is Qo'noS
   romulan: 'Jolan\'tru', // home planet is Romulus
   'federation standard': 'hello' // home planet is Earth
 };
 
+var klingonGreeting;
+var romulanGreeting;
+var humanGreeting;
+
 // TODO: define a constructor that creates objects to represent
 // sentient beings. They have a home planet, a language that they
 // speak, and method (that you'll place on the prototype) called
 // sayHello.
-
-function SentientBeing () {
-  // TODO: specify a home planet and a language
-  // you'll need to add parameters to this constructor
+function SentientBeing(hmPlanet, lang) {
+  this.homePlanet = hmPlanet;
+  this.language = lang;
+  klingonGreeting = hello.klingon;
+  romulanGreeting = hello.romulan;
+  humanGreeting = hello['federation standard'];
+
+  if (lang === 'klingon') {
+    this.myHello = klingonGreeting;
+  }
+  if (lang == 'romulan') {
+    this.myHello = romulanGreeting;
+  }
+  if (lang == 'federation standard') {
+    this.myHello = humanGreeting;
+  }
```

Comment:
good! Glad you got it working. If you want to work on this further, think about what you can do with bracket notation and the hello object.